x-pack/legacy/plugins/watcher/public/services/watches/watches.js

```diff
@@ -4,23 +4,18 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import chrome from 'ui/chrome';
-import { ROUTES } from '../../../common/constants';
 import { Watch } from 'plugins/watcher/models/watch';
+import { npStart } from 'ui/new_platform';
+const { http } = npStart.core;
 
-export class WatchesService {
-  constructor($http) {
-    this.$http = $http;
-    this.basePath = chrome.addBasePath(ROUTES.API_ROOT);
-  }
-
+export const watches = {
   getWatchList() {
-    return this.$http.get(`${this.basePath}/watches`)
-      .then(response => response.data.watches)
-      .then(watches => watches.map(watch => {
-        return Watch.fromUpstreamJson(watch);
-      }));
-  }
+    return http.get('/watches')
```

Comment:
@eliperelman This does not appear to be appending the base path :-(

(Nor the `http.post` below)

Reply:
Resolved in new commit